### DISCUSSION TITLE
Updating from 1.5.1

### DISCUSSION THREAD

**Discussion by gorman42** [DigitalRuby/IPBan#245] [Q&A] [answered]
Hi! I have an old installation of IPBan, 1.5.1. How am I supposed to update? Else, how can I uninstall 1.5.1 cleanly, so as to install 1.8.0 in the right way?

Thanks for the help.

**Comment by jjxtra** [comment]
> Hi! I have an old installation of IPBan, 1.5.1. How am I supposed to update? Else, how can I uninstall 1.5.1 cleanly, so as to install 1.8.0 in the right way?
> 
> 
> 
> Thanks for the help.

There should be instructions on uninstalling in the script you used to install.

**Reply by gorman42** [reply]
> There should be instructions on uninstalling in the script you used to install.

Nope, I checked before writing. The REAME.md file contains only installation instructions, as far as I can see.

**Comment by jjxtra** [comment, marked as answer]
See comment in this script on line 14. You'll have to download the script to your computer.

https://github.com/DigitalRuby/IPBan/blob/master/IPBanCore/Windows/Scripts/install_latest.ps1

**Reply by gorman42** [reply]
Ok, I could not execute the script due to it lacking digital signature or something.

```
.\install_latest.ps1 : File D:\downloads\install_latest.ps1 cannot be loaded. The file D:\downloads\install_latest.ps1
is not digitally signed. You cannot run this script on the current system. For more information about running scripts
and setting execution policy, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170
```

But I browsed the script and isolated the commands needed to uninstall the old version (stopping the service, deleting the service). I then ran the install latest script as per the current read.me instructions and it appears to be installed correctly.
